06_PRC_Handson_User-Training.md

```diff
@@ -340,12 +387,12 @@ However, if you want to remove all existing metadata on an object at once, then
 
 ### Atomic operations on metadata
 
-The PRC allows a group of metadata add and remove operations to be performed transactionally, within a single call to the server. This does mean you can apply atomic operations on metadata. In other words, you can add more than one AVU and also remove metadata at the same operation. One important thing to know is that the list of operations will be applied in the order given.
+The PRC allows you to add and remove metadata in sequence with a single operation, i.e., a single call to the server. The `apply_atomic_operations()` method takes a series of `AVUOperation` objects as arguments and implements them in the order given. To the `AVUOperation()` call we specify the kind of operation ("remove" or "add") and the AVU involved as an `iRODSMeta` object. Therefore, we need to import both the `iRODSMeta` class (already imported above) and `AVUOperation`.
```

Comment:
Now that I'm thinking of it, maybe it's a good idea to mention the main advantage of the atomic operations: because you add/remove multiple avu's in a single server call, it's much **faster.**